src/components/screens/Authenticator/Authenticator.interactor.vue

```diff
@@ -54,6 +58,16 @@ export default {
       },
       immediate: true,
     },
+
+    isRemembered: {
+      handler(newVal) {
+        this.$emit('update:is-remembered', newVal);
```

Comment:
It's very strange. Do we really need `remembered` and `remember`? It's very difficult for reading code